docs/how_to_add_test_for_rector_rule.md

```diff
@@ -0,0 +1,119 @@
+# How to Add Test for Rector Rule
+
+## 1. Detect the Rector Rule
+
+Run Rector only on 1 directory, or better 1 file.
+
+```bash
+vendor/bin/rector process compiler/src/PhpScoper/StaticEasyPrefixer.php
+```
+
+See "Applied rules" under the diff:
+
+![Applied Rules](/docs/images/docs_applied_rules.png)
+
+Our rule in this case is: `Rector\SOLID\Rector\Class_\FinalizeClassesWithoutChildrenRector`
+
+## 2. Detect the Minimal File
+
+Usually the Rector diff output is long and contains many other errors related to other rules. It's a mess, we can't use that for test fixture. We need to find **1 responsible line**.
+
+The best way is to copy the file to local code, e.g. `app/SomeFile.php` a put only the broken line there.
+
+In our case, all we need is:
+
+```php
+class StaticEasyPrefixer
```

Comment:
I think it should be mentioned that this is because `FinalizeClassesWithoutChildrenRector` adds `final` to the class, otherwise it's not clear...

Reply:
I mean, the image can be seen in the image, but for newbies it's still a bit not clear, and I guess you're targetting newbies with this doc